lib/langchain/llm/google_palm.rb

```diff
@@ -19,9 +19,9 @@ class GooglePalm < Base
       embeddings_model_name: "embedding-gecko-001"
     }.freeze
     LENGTH_VALIDATOR = Langchain::Utils::TokenLength::GooglePalmValidator
-    ROLE_MAPPING = {
-      "human" => "user"
-    }
+    COMPLETION_RESPONSE_PATH = ["candidates", 0]
+    ROLE_KEY = :author
```

Comment:
I don't think we need to have these role mappings at all. This is only needed in the context of conversation and `Conversation` class is already aware if the message was sent by the user it is a response from LLM.